en/docs/learn/configuring-inbound-provisioning.md

```diff
@@ -0,0 +1,53 @@
+# Configuring Inbound Provisioning 
+
+## Introduction
+
+Provisioning is the process of creating, maintaining and deleting digital identities (accounts) for users of a system(s) and linking appropriate rights to identities in the form of rules and roles. 
+
+Inbound provisioning provisions users or groups into the WSO2 Identity Server by an external application. These external applications are referred to as service providers. WSO2 Identity Server supports the SCIM API and SOAP-based Web service API standards for inbound provisioning.
```

Comment:
into WSO2 Identity Server

Reply:
Fixed in https://github.com/wso2/docs-is/pull/982 commit fa76065